File: java/ReverseInteger.java
public class ReverseInteger {
    public int reverse(int x) {
        int t = 0;
        while (x != 0) {
            t = t * 10 + x % 10;
            x /= 10;
        }
        return t;
    }

    public static void main(String[] args) {
        ReverseInteger solution = new ReverseInteger();
        int x;
        x = -123;
        System.out.println(solution.reverse(x));
    }
}

File: cc/reverse_integer.cc
class ReverseInteger {
public:
    int reverse(int x) {
        int res = 0;
        while (x != 0) {
            res = res * 10 + x % 10;
            x /= 10;
        }
        return res;
    }
};

int main(int argc, char *argv[]) {
    return 0;
}
